go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/ory/hydra
 
 go 1.16
 
+require github.com/ory/xnew v0.0.0-00010101000000-000000000000
```

Comment:
Use this instead pls:

https://github.com/ory/x/blob/36d472c791c1376b247a90c7e62697139830e306/go.mod#L13